mindjunkies/payments/migrations/0002_balance_balancehistory.py

```
@@ -0,0 +1,54 @@
+    operations = [
+        migrations.CreateModel(
+            name='Balance',
+            fields=[
+                ('id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False, unique=True)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('deleted_at', models.DateTimeField(blank=True, null=True)),
+                ('amount', models.DecimalField(decimal_places=2, default=0, max_digits=10)),
+                ('last_updated', models.DateTimeField(auto_now=True)),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='balance', to=settings.AUTH_USER_MODEL)),
```

Ruff (0.8.2)

Line too long (145 > 119)

(E501)